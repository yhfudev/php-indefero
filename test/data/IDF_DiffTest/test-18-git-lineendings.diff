diff --git a/lineendings b/lineendings
index 7c2b7ec..9c59944 100644
--- a/lineendings
+++ b/lineendings
@@ -1,3 +1,4 @@
+Unix, again
 Windows
 Unix
 Old MacintoshNothing
\ No newline at end of file
