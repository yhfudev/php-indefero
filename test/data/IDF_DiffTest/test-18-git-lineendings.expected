<?php return array (
  'lineendings' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '',
          1 => '1',
          2 => "Unix, again\n",
        ),
        1 => 
        array (
          0 => '1',
          1 => 2,
          2 => "Windows\r\n",
        ),
        2 => 
        array (
          0 => 2,
          1 => 3,
          2 => "Unix\n",
        ),
        3 => 
        array (
          0 => 3,
          1 => 4,
          2 => "Old Macintosh\rNothing",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '3',
        ),
        1 => 
        array (
          0 => '1',
          1 => '4',
        ),
      ),
    ),
  ),
);